app/views/clinical_resource_hubs/show.html.erb

```diff
@@ -1,8 +1,34 @@
-<div>
-  <%# debugger %>
-  VISN NUmber: <%= params["id"] %>
-  <div>
-  VISN_ID: <%= @crh.visn_id %>
+<% provide :main_classes, 'bg-gray-2' %>
+<% provide :body_classes, 'bg-gray-2' %>
+<% provide :footer_classes, 'bg-gray-2' %>
+
+<% visn_number = params["number"]  %>
+<% visn_id = @crh.visn_id %>
+
+<% provide :head_tags do %>
+  <%= javascript_include_tag '_practice_card_utilities', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
+
+    <%= render partial: 'crh_show', formats: [:js] %>
+    <%# set visnNumber before loading the crh/show file %>
+    var visnNumber = <%= visn_number %>;
+  <% end %>
+  <%= javascript_include_tag 'clinical_resource_hubs/crh_show', 'data-turbolinks-track': 'reload' %>
+<% end %>
+
+<div id="crh-show" class="grid-col-12" data-reload="true">
+  <section class="usa-section padding-y-0 grid-col-12" id="dm-crh-show">
+    <div class="grid-container position-relative grid-col-12">
+      <h1 class="margin-top-0 margin-bottom-3">
+        <%= @crh.official_station_name %>
+      </h1>
+      <div class="grid-row grid-gap-3 margin-bottom-10">
+        <div class="grid-col-12 line-height-26"> This Clinical Resource Hub (CRH) serves veterans in <%= @states_str %>. It has created <%= @practices_created_by_crh.count %> innovations and has adopted <%= @practices_adopted_by_crh.count %> innovations.
```

Comment:
We aren't taking into account a single origin facility/adoption here. If the CRH has just one adoption, it reads "adopted 1 innovations."

Reply:
Got it - good catch.